gpflow/ci_utils.py

```diff
@@ -18,7 +18,14 @@
 
 
 def is_continuous_integration():
-    return os.environ.get('CI', None) is not None
+    """
+    If the environment variable `DOCS` is set, always returns False.
+    Otherwise, returns `True` if notebooks are executed on CI, else `False`.
+    """
+    if 'DOCS' in os.environ:
```

Comment:
would it make sense to change this to `"READTHEDOCS"` (to be consistent with what RTD sets already)? (I'm happy with it as it is, too)

Reply:
This is actually a environment variable used by circleci